client/src/store/modules/eventType/normalizer.js

```diff
@@ -4,7 +4,6 @@ export const eventTypeMapper = EventType => ({
     name: EventType.name,
     description: EventType.description || '',
     internalNote: EventType.internal_note,
-    location: EventType.location,
     locationType: locationTypeMapper(EventType.location_type),
```

Comment:
In event presenter this variables is location. I don't see location_type in event presenter

Reply:
Event?
There is EventType presenter